go.mod

```diff
@@ -5,7 +5,8 @@ go 1.16
 require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
-	github.com/gardener/external-dns-management v0.7.20
+	github.com/gardener/controller-manager-library v0.2.1-0.20210519160602-fb05e73bad23
```

Comment:
Can we prevent importing this?
![image](https://user-images.githubusercontent.com/19169361/119090515-01f2df00-ba0c-11eb-8a5d-e511c934c45a.png)

Reply:
The idea was to fetch the DNSAnnotation CRD directly from the `external-dns-management` dependency. For this purpose the controller-manager-library is mandatory. The alternative is to embed the CRD as a text file, but then it needs to be updated if there are any changes.

Reply:
Reverted to original solution with embedding the CRD in version v1beta1